Learning/Tutorials/Official-Bevy-Examples.toml

```diff
@@ -0,0 +1,4 @@
+name = "Official Bevy Examples"
```

Comment:
I think ultimately we should have a "bevy asset toml format validator" that runs on every entry. That way we never get into a bad state.

Reply:
(not saying we need that right away though)

Reply:
yes, once happy with the format I'll change the update tool in bevy-website to have two mains, one for transformation and one for validation, and will use it here

Reply:
added in tracking issue